src/components/common/ProductBox/ProductBoxContainer.js

```diff
@@ -0,0 +1,14 @@
+import { connect } from 'react-redux';
+import ProductBox from './ProductBox';
+import { setCompare, getCount } from '../../../redux/compareRedux';
+
+const mapStateToProps = state => ({
+  count: getCount(state),
+  compare: state.compare,
```

Comment:
Please use a selector for extracting data from the store. You can create a new one if needed.